metadata.json

```diff
@@ -9,26 +9,26 @@
   "issues_url": "http://projects.theforeman.org/projects/katello/issues",
   "dependencies": [
     {
-      "name": "puppetlabs-stdlib",
+      "name": "puppetlabs/stdlib",
       "version_requirement": ">= 1.0.0"
     },
     {
-       "name": "puppetlabs-apache",
+       "name": "puppetlabs/apache",
        "version_requirement": ">= 1.0.0 < 2.0.0"
     },
     {
-       "name": "puppetlabs-mongodb"
+       "name": "puppetlabs/mongodb"
     },
     {
-       "name": "theforeman-foreman",
+       "name": "theforeman/foreman",
        "version_requirement": ">= 1.5.0"
     },
     {
-       "name": "theforeman-concat_native",
+       "name": "theforeman/concat_native",
        "version_requirement": ">= 1.3.0"
     },
     {
-       "name": "katello-qpid",
+       "name": "katello/qpid",
        "version_requirement": ">= 0.1.0"
     }
   ]
```

Comment:
puppet still has this bug: https://tickets.puppetlabs.com/browse/FORGE-52

Meaning that "puppet module install" will fail to find any of the dependencies